SpectralCubeReprojectMaskExample_v3.ipynb

```diff
@@ -0,0 +1,1546 @@
+    "\n",
+    "In this case, it's redundant and we could have just used cube1's beam, but this is the more general approach:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "Beam: BMAJ=$1.29719604986604^{''}$ BMIN=$1.04247149438736^{''}$ BPA=$82.95313553702^\\circ$"
+      ],
+      "text/plain": [
+       "Beam: BMAJ=1.29719604986604 arcsec BMIN=1.04247149438736 arcsec BPA=82.95313553702 deg"
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import radio_beam\n",
+    "common_beam = radio_beam.commonbeam.common_2beams(radio_beam.Beams(beams=[cube1vel.beam, cube2vel.beam]))\n",
+    "common_beam"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We convert to Kelvin to ensure the units are preserved"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: PossiblySlowWarning: This function (<function BaseSpectralCube.to at 0x2ad7b51e3730>) requires loading the entire cube into memory and may therefore be slow. [spectral_cube.utils]\n",
+      "WARNING: PossiblySlowWarning: This function (<function BaseSpectralCube.convolve_to at 0x2ad7b51e51e0>) requires loading the entire cube into memory and may therefore be slow. [spectral_cube.utils]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "SpectralCube with shape=(75, 420, 420) and unit=K:\n",
+       " n_x:    420  type_x: RA---SIN  unit_x: deg    range:   266.537002 deg:  266.551600 deg\n",
+       " n_y:    420  type_y: DEC--SIN  unit_y: deg    range:   -28.711371 deg:  -28.698569 deg\n",
+       " n_s:     75  type_s: VRAD      unit_s: km / s  range:      -43.509 km / s:     104.685 km / s"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cube2vel_spatialspectralsmooth = cube2vel_spectralresample.to(u.K).convolve_to(common_beam)\n",
+    "cube2vel_spatialspectralsmooth"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step 6. Reprojection\n",
+    "Now, finally, we can do the reproject step:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "4.3.dev1788+ga3263b6\n",
+      "0.8.dev31+g9f8ec7b\n"
+     ]
+    }
+   ],
+   "source": [
+    "import astropy\n",
+    "print(astropy.__version__)\n",
```

Comment:
Move version checks to the beginning.

@abulatek -- is the dev version of astropy still needed?

Reply:
I tested by downgrading from 4.3.dev1788+ga3263b6 to 4.3.1, and the reproject step works, so I don't think we need the dev version!